tests/test_utils.py

```diff
@@ -76,6 +76,19 @@ def testParseTimeInterval2a(self):
         interval = parse_time_interval("hours=1, seconds = 2")
         self.assertEqual(timedelta(hours=1, seconds=2), interval)
 
+    def testParseTimeInterval2b(self):
+        interval = parse_time_interval("1 hour, 1 second")
+        self.assertEqual(timedelta(hours=1, seconds=1), interval)
+
+        interval2 = parse_time_interval("1 hour, 10 milliseconds")
+        self.assertEqual(timedelta(hours=1, milliseconds=10), interval2)
+
+        interval3 = parse_time_interval("1 hour, 10 microseconds")
+        self.assertEqual(timedelta(hours=1, microseconds=10), interval3)
+
+        interval4 = parse_time_interval("1 year, 4 weeks")
+        self.assertEqual(timedelta(weeks=56), interval4)
```

Comment:
from readability perspective it's better to parametrize this test as described [here](https://stackoverflow.com/questions/32899/how-do-you-generate-dynamic-parameterized-unit-tests-in-python) for instance

Reply:
good suggestion

Reply:
but requires migrating tests from unit test to pytest 

Currently tests use pytest in unittest compatibility mode so parameterize does not work. Dont want additional external lib dependency for minor change so simply rewriting as for next loop

Will use simple for next loop with array of tuples for readability